engine/src/background.rs

```diff
@@ -79,7 +79,7 @@ impl SvgTangibleObject for Background {
                 let mut defs = Element::new("defs");
 
                 let mut linear = Element::new("linearGradient");
-                linear.set_attr("id", "background");
+                linear.set_attr("id", format!("background-{}", id));
```

Comment:
nitpick: format might be the slowest method on every language if we are doing simple string concat https://github.com/hoodie/concatenation_benchmarks-rs#format_macro

Reply:
Second this. I filed an issue `FL-33` regarding performance optimizing.